frontend/ui/widget/filechooser.lua

```diff
@@ -128,9 +128,15 @@ function FileChooser:genItemTableFromPath(path)
         else
             sstr = string.format("%d B", file_size)
         end
+        -- show files with a .sdr in bold
+        local bold = nil
+        if full_path:match("%.") and lfs.attributes(full_path:match("(.*)%.")..".sdr", "mode") == "directory" then
```

Comment:
Better use `getSidecarDir` method there to get the sidecar directory. You can import DocSettings at the beginning of this module, then use it here:

```lua
local DocSettings = require("docsettings")
...

        if lfs.attributes(DocSettings:getSidecarDir(full_path), "mode") == "directory" then
        end
```